packages/sysutils/open-vm-tools/package.mk

```diff
@@ -24,7 +24,6 @@ PKG_CONFIGURE_OPTS_TARGET="--disable-docs \
                            --without-x \
                            --without-xerces \
                            --without-icu \
-                           --without-procps \
```

Comment:
no bump to `10.3.10` for open-vm-tools ?

Reply:
Added